app/assets/javascripts/models/data/project.js.coffee

```diff
@@ -79,6 +79,7 @@ ns.Project = do (ko,
         dataType: 'json'
         url: '/data/options'
         data: params
+#        method: 'post'
```

Comment:
My bad. I must have commented out during the testing... I will correct it later today.